.circleci/config.yml

```diff
@@ -221,6 +225,25 @@ jobs:
           name: Type Checking
           command: yarn run type-check:v3
 
+  legacy algoliasearch v4:
+    <<: *defaults
+    steps:
+      - checkout
+      - *attach_workspace
+      - *install_yarn_version
+      - *restore_yarn_cache
+      - *run_yarn_install
+      - run:
+          name: Update dependencies
+          command: |
+            ./scripts/legacy/downgrade-algoliasearch-v4.js
+      - run:
+          name: Unit & Integration tests
+          command: yarn run test
```

Comment:
we need to store the test results like in unit tests

Reply:
Done and everything passes ! Thanks !